sharepoint/sharepoint-server-ps/sharepoint-server/Set-SPWebApplication.md

```diff
@@ -222,14 +277,31 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Confirm
-Prompts you for confirmation before executing the command.
-For more information, type the following command: `get-help about_commonparameters`
+### -Identity
+Specifies the name or URL of the Web application.
+
+The type must be a valid name, in the form WebApplication-1212, or URL, in the form http://server_name.
+
+```yaml
+Type: SPWebApplicationPipeBind
+Parameter Sets: (All)
+Aliases: 
+Applicable: SharePoint Server Subscription Edition
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -NotProvisionGlobally
+Skips the Web application provision process.
```

Comment:
Correct me if I'm wrong, but doesn't this parameter simply provision the Web Application on the specific server the switch is run on? I.e., I would say something like "Only provisions the Web Application on the server the switch is specified on rather than all servers running the Foundation Web Service service instance".

Reply:
@TroyStarr Could you help to take a look and give suggestions on this?

Reply:
I agree with @tseward - we should clarify this to say something like "Only provisions the web application on the local server with the changes specified by this cmdlet. Web applications on other servers in the farm will not be provisioned with these changes."